.github/workflows/test-sui.yaml

```diff
@@ -35,25 +35,23 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - name: Setup Node.js
-        uses: useblacksmith/setup-node@v5
-        with:
-          node-version: 18.x
-          cache: 'npm'
-
-      - name: Install
-        run: npm ci
-
-      - name: Read SUI_VERSION from JSON
-        id: read-json
+      - name: Prepare version.json file
         run: |
-          SUI_VERSION=$(jq -r '.SUI_VERSION' sui/version.json)
-          echo "SUI_VERSION=${SUI_VERSION}" >> $GITHUB_ENV
+          curl -sL https://raw.githubusercontent.com/axelarnetwork/axelar-cgp-sui/v0.6.0/version.json -o version.json
```

Comment:
get the `version.json` from the installed node_modules/axelar-cgp-sui instead? otherwise we end up having to manage this url instead

Reply:
we don't include `version.json` in the published package. we cannot avoid edit this file completely because of the `uses` syntax.

Reply:
let's include the version.json in axelar-cgp-sui release?